.github/workflows/auto-author-assign.yml

```
@@ -0,0 +1,15 @@
+on:
+  pull_request_target:
+    types: [ opened, reopened ]
+
+permissions:
+  pull-requests: write
+
+jobs:
+  assign-author:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: toshimaru/auto-author-assign@v2.1.0
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)